### DISCUSSION TITLE
Block Protocol

### DISCUSSION THREAD

**Discussion by jcklpe** [uswds/uswds#4601] [General]
Not necessarily design system related but I think this project sounded interesting and worth checking out. 

Intro post:
https://www.joelonsoftware.com/2022/01/27/making-the-web-better-with-blocks

Main website:
https://blockprotocol.org/

Blocks as a concept in stuff like Wordpress aligns heavily with components. Might be worth checking out the protocol for some ideas.

**Comment by jcklpe** [comment]
It's concept of blocks as structured data aligns with the "design tokens as design system data layer" stuff I have mentioned in the past.

**Comment by mejiaj** [comment]
This is pretty cool, thank you for sharing!

**Comment by jcklpe** [comment]
When looking this up again I accidentally ran into this and it seems even more relevant to USWDS work: https://www.govstack.global/govstack-release-1-1-0/
